README.md

```diff
@@ -1,30 +1,32 @@
 # Bryntum Salesforce showcase
 
-This application demonstrates Bryntum components in Salesforce environment.
+This application demonstrates how Bryntum components can be used in a Salesforce environment.
 
-## Design
+## Application structure
 
-This app creates 6 static resources, 6 Lightning Web Components and a Lightning `Bryntum` App. There is an LWC
-for every Bryntum product: Grid, Scheduler, SchedulerPro, Gantt, Taskboard and Calendar. Each LWC uses
-bundle-specific static resource. By default app uses dummy resource, to enable LWC you need to create
-proper resource (see below for instructions)
+The Bryntum Salesforce showcase application creates six static resources, six Lightning Web Components (LWCs), and a Lightning `Bryntum` app.
 
-## Setup
+An LWC is provided for each Bryntum product: Grid, Scheduler, Scheduler Pro, Gantt, Task Board, and Calendar. Each LWC uses a bundle-specific static resource.
```

Comment:
```suggestion
An LWC is provided for each Bryntum product: Grid, Scheduler, Scheduler Pro, Gantt, TaskBoard, and Calendar. Each LWC uses a bundle-specific static resource.
```

Reply:
there's a big mix of TaskBoard and Task Board throughout the product pages, docs, etc. We've been standardizing everything to "Task Board" so far, but let me know if it should be the other way around?

Reply:
Discussed on Slack and we agreed on two words.

![screenshot 2025-01-21 at 18 14 51@2x](https://github.com/user-attachments/assets/8039b227-442f-4639-8674-b1bdb8516690)